### PR TITLE
[MODULAR][FIX] Cryo now saves modular persistence

### PR DESCRIPTION
**Pull request by softcerv**
## About The Pull Request
This PR makes it so that the cryopods save persistence when someone enters rather than when someone is removed. Because the cryopods only saved once someone is removed, they would return early due to the mob missing a mind, by saving when the mob enters, persistence is able to access the mind to save to.

## How This Contributes To The Skyrat Roleplay Experience
Bugfixes are good for RP :)

## Proof of Testing

<details>
<summary>Screenshots/Videos</summary>

compile
image

Before entering cryo
image

image

after respawning after being cryo'd
image  
</details>

## Changelog

:cl:
fix: cryo now saves modular persistence
/:cl:

### CODE REVIEW

**Comment by softcerv** [comment]
https://github.com/Skyrat-SS13/Skyrat-tg/issues/19476

File: modular_skyrat/modules/cryosleep/code/cryopod.dm [hunk 1]
```diff
@@ -221,6 +221,10 @@ MAPPING_DIRECTIONAL_HELPERS(/obj/machinery/computer/cryopod, 32)
 			if(mob_occupant.mind)
 				stored_rank = mob_occupant.mind.assigned_role.title
 
+		var/mob/living/carbon/human/human_occupant = occupant
+		if(human_occupant)
+			human_occupant.save_individual_persistence()
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/cryosleep/code/cryopod.dm:226]: Does this handle the case where someone is added to cryo at the point where they don't have a mind anymore? Say, through admin intervention (which isn't all that uncommon).

**Review by GoldenAlpharex — CHANGES_REQUESTED** [review]

**Comment by softcerv** [comment]
Before:

image

After:

image

image

File: modular_skyrat/modules/cryosleep/code/cryopod.dm (hunk 1 above)
Comment (softcerv) [reply modular_skyrat/modules/cryosleep/code/cryopod.dm:226]: Fixed

**Review by GoldenAlpharex — APPROVED** [review]
Looks great to me, thank you.

**Pull request merged**